### DISCUSSION TITLE
Return only a number.

### DISCUSSION THREAD

**Discussion by Brncray** [msiemens/tinydb#499] [Q&A] [answered]
I am using this for an economy bot on discord. How would I return only a number? 
`   
 results = db.search(User.id == ctx.options.member)
    if results == []:
        await ctx.respond("User doesn't exist... adding to database.")
        db.insert({"id": ctx.options.member, "bal": 0})
    else:
        await ctx.respond(results)
`
![image](https://user-images.githubusercontent.com/57870234/211207870-40872ca7-0afb-4ad3-aed5-54816b8ed66c.png)

I want to only return the name and the balance, take away the brackets.

**Comment by msiemens** [comment, marked as answer]
Would something like this fork for you?

```python
results = db.search(User.id == ctx.options.member) 

if results == []:
    await ctx.respond("User doesn't exist... adding to database.")
    db.insert({"id": ctx.options.member, "bal": 0})
else:
    await ctx.respond(results[0].bal)
```

**Reply by Brncray** [reply]
I got the error, `AttributeError: 'Document' object has no attribute 'bal'`.

**Reply by msiemens** [reply]
Ah, that's my fault. Try this instead:

```python
results = db.search(User.id == ctx.options.member) 

if results == []:
    await ctx.respond("User doesn't exist... adding to database.")
    db.insert({"id": ctx.options.member, "bal": 0})
else:
    await ctx.respond(results[0]['bal'])
```

**Reply by Brncray** [reply]
This worked, thank you :)
